harvard-university-for-the-creative-arts.csl

```diff
@@ -123,7 +138,18 @@
         <group>
           <text variable="genre"/>
           <text variable="medium" prefix="[" suffix="]"/>
+          <text variable="dimensions" suffix="."/>
           <text variable="note" prefix=" " suffix="."/>
+          <text variable="archive_location" prefix=" "/>
+        </group>
+        <text variable="archive" prefix=": " suffix="."/>
+      </else-if>
+      <else-if type="speech personal_communication" match="any">
+        <group delimiter=" " prefix="[">
+          <text variable="genre" suffix=" "/>
+          <text variable="note"/>
+          <text variable="event-place" suffix=":"/>
+          <date form="text" variable="issued" suffix="]"/>
```

Comment:
@ibadger, this square bracket suffix should be on the group element. In general, it's best to move as many delimiting punctuation to group elements.